### PR TITLE
Clean up volumes and ports in Dockerfile.

### PR DESCRIPTION
**Pull request by fryfrog**
Re-do ports and volumes in a more "docker" way, add #comments about what each port does from https://help.ubnt.com/hc/en-us/articles/217875218-UniFi-Video-Ports-Used

### CODE REVIEW

**Comment by fryfrog** [comment]
The log volume change only applies to 3.9.2+, so should wait until that is in.

**Comment by fryfrog** [comment]
This also shouldn't go in until 3.9.2 works *and* this is tested w/ it. :/

File: Dockerfile [hunk 1]
```diff
@@ -25,11 +25,41 @@ RUN apt-get update && \
...
+EXPOSE 7445/tcp
+
+# Video over HTTPS
+EXPOSE 7446/tcp
+
+# RTSP via the controller
+EXPOSE 7447/tcp
+
+# RTMP via the controller
+EXPOSE 1935/tcp
```
Comment (fryfrog) [inline Dockerfile]: I was *sure* streaming would be udp, but the internet says rtmp is TCP! :/

File: Dockerfile [hunk 2]
```diff
@@ -25,11 +25,41 @@ RUN apt-get update && \
...
+EXPOSE 7443/tcp
+
+# Video over HTTP
+EXPOSE 7445/tcp
+
+# Video over HTTPS
+EXPOSE 7446/tcp
+
+# RTSP via the controller
+EXPOSE 7447/tcp
```
Comment (fryfrog) [inline Dockerfile]: Streaming not using UDP, isn't that crazy? But unifi's document for ports says TCP.

File: README.md [hunk 3]
```diff
@@ -24,6 +24,8 @@ docker run \
         -p 7447:7447 \
         -p 7080:7080 \
         -p 6666:6666 \
+        -p 1935:1935 \
+        -p 7447:7447 \
```
Comment (i8beef) [inline README.md]: This (7447) was already exposed, 7444 was the one that would be added.

**Comment by i8beef** [comment]
Looks good for closing #59. Should be merged in with 3.9.2, or to merge earlier, put additional volume in for /var/log/unifi-video temporarily (to be removed when 3.9.2 is put together).

**Comment by fryfrog** [comment]
So if we fix it here, what happens to people who update and *don't* change their `docker run`? :/

**Pull request merged**